### PR TITLE
Added testing for Django 3.2.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: tests/runtests.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from django.conf import settings
 settings.configure(
     DATABASES={'default': {'ENGINE': 'django.db.backends.sqlite3'}},
+    DEFAULT_AUTO_FIELD='django.db.models.BigAutoField',
```
Comment (atodorov) [inline tests/runtests.py:18]: Why is this different from apps.py ?

Comment (atodorov) [reply tests/runtests.py:18]: And why is it needed at all ?

**Review by atodorov — CHANGES_REQUESTED** [review]
Overall looks good but CI fails and needs a few explanations.

**Comment by claudep** [comment]
The test failures is weird, indeed! The global apps symbol is `django_comments.apps` instead of the imported `django.apps.apps`. @felixxm, any idea what's going on? I bisected the problem to https://github.com/django/django/commit/3f2821af6bc4

**Comment by felixxm** [comment]
> The test failures is weird, indeed! The global apps symbol is `django_comments.apps` instead of the imported `django.apps.apps`. @felixxm, any idea what's going on? I bisected the problem to [django/django@3f2821a](https://github.com/django/django/commit/3f2821af6bc4)

It's a side-effect of an `import_module('django_comments.app')` [call](https://github.com/django/django/blob/90ddf46ef7b3d775b124d81e1846bec7961c7f1f/django/apps/config.py#L124). In older versions of Django, we have the same side-effect with `default_app_config = 'django_comments.apps.DjangoCommentsAdminConfig'` defined in `django_comments/__init__.py` so it's not a regression, IMO.

We could add a warning to the docs that importing `django.apps` in `__init__.py` is a bad idea :thinking: 

(\cc @aaugustin).

File: tests/runtests.py (hunk 1 above)
Comment (felixxm) [reply tests/runtests.py:18]: > Why is this different from apps.py ?

Using `BigAutoField` in `DjangoCommentsAdminConfig` would generate migrations. In tests we can use `BigAutoField` without any side-effects. See [release notes](https://docs.djangoproject.com/en/3.2/releases/3.2/#customizing-type-of-auto-created-primary-keys) for more details.

**Comment by felixxm** [comment]
> We could add a warning to the docs that importing `django.apps` in `__init__.py` is a bad idea

OK, it's already [there](https://docs.djangoproject.com/en/dev/ref/applications/#configuring-applications-ref):

> If your code imports the application registry in an application’s `__init__.py`, the name apps will clash with the apps submodule. The best practice is to move that code to a submodule and import it. A workaround is to import the registry under a different name:

File: HISTORY.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ Unreleased
   (more about that in the documentation) with an appropriate
   ``object_pk`` field.
 * Confirmed support for Python 3.9.
-
+* Confirmed support for Django 3.2.
```
Comment (claudep) [inline HISTORY.rst:19]: As the `apps` change was required, should we replace `Confirmed` by `Added`?

Comment (felixxm) [reply HISTORY.rst]: Updated.

**Comment by claudep** [comment]
Thanks a lot!

**Pull request merged**